app/api/items/get_item_can_request_help_to_groups.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Get permissions has_can_request_help_to for an item
+  Background:
+    Given there are the following groups:
+      | group               | parent  | members               |
+      | @InaccessibleSchool |         | @InaccessibleGroup    |
+      | @School             |         | @TeacherGroup         |
+      | @TeacherGroup       |         | @Teacher              |
+      | @ClassParent        |         | @Class                |
+      | @Class              | @School | @Student,@HelperGroup |
+    And @Teacher is a manager of the group @Class and can watch its members
+    And there are the following items:
+      | item                 | parent    | type    | request_help_propagation |
+      | @Chapter1            |           | Chapter |                          |
+      | @Chapter2            |           | Chapter |                          |
+      | @Chapter3            |           | Chapter |                          |
+      | @Chapter4            |           | Chapter |                          |
+      | @Item1               | @Chapter1 | Task    |                          |
+      | @Item2               | @Chapter2 | Task    | true                     |
```

Comment:
Not clear to me what `request_help_propagation` column means on an item (as it is a property of a relation). Don't you want to call it "request_help_propagation_with_parent"? Maybe a bit long... :-/
And what does it mean at L20 when there is a parent but no value?

Reply:
fixed for `request_help_propagation `. Definition of item relations was added for `viewPermissions`, so it's been reused.

No value for `request_help_propagation` means: "this value wasn't necessary to write the test". So that the property is not relevant to the test. It makes reading the test (at least for me), easier because I don't have to focus on information that is not relevant.